docs/crds/hostrule.md

```diff
@@ -309,6 +309,105 @@ The `networkSecurityPolicy` setting, in addition to any other parameters provide
 1. This property is available only in HostRule `v1beta1` schema definition.
 2. The HostRule CRD is not aware of the misconfigurations if it is applied to Child VS while it is being created, therefore the HostRule will be `Accepted` nonetheless. AKO will print warning message regarding this.
 
+#### Enable Regular Expression in path
+When at least one of the paths in Kubernetes Ingress or OpenShift Route is a regular expression, hostrule should be defined with the `useRegex` field. It is an optional field.  
+
+Based on if a hostrule is defined with the `useRegex` field, AKO will handle the ingress/route in the following ways.
+
+1. If a hostrule is defined with the useRegex field set to **True** for a given host, then all paths defined for that host, regardless of what Ingress/Route they are defined on, will be considered as case-insensitive regular expressions. Avi will create rules for each path for regex pattern matching.
+
+        fqdn: test-avi.vmware.com
+        fqdnType: Exact
+        useRegex: True
+
+   Avi will always follow the longest match approach to match the path in the incoming request URL, against the available regex paths for the host.
+
+        apiVersion: networking.k8s.io/v1
+        kind: Ingress
+        metadata:
+          name: test-ingress
+        spec:
+          rules:
+          - host: test-avi.vmware.com
+            http:
+              paths:
+              - backend:
+                  service:
+                    name: my-app-0
+                    port:
+                      number: 80
+                path: /bar.*
+                pathType: ImplementationSpecific
+              - backend:
+                  service:
+                    name: my-app-1
+                    port:
+                      number: 80
+                path: /bar/foo
+                pathType: ImplementationSpecific
+          tls:
+          - hosts:
+            - test-avi.vmware.com
+            secretName: ingress-secret
+
+   For the above Ingress definition, if we create a hostrule with `useRegex` set to True, then :  
+   a) A request that comes on the URL **https://test-avi.vmware.com/bar/foo** will be routed to backend **my-app-1**.  
+   b) A request that comes on the URL **https://test-avi.vmware.com/bar/baz** will be routed to backend **my-app-0**.
+
+2. If a hostrule is defined with the useRegex field set to **False**, or no hostrule is defined for a host and the user specifies paths that are regular expressions in Ingress/Route for the same host, then Avi will not consider the paths as regular expressions and will create rules that will match the whole path instead of regex pattern matching.
+
+***Note***
+1. This property is available only in HostRule `v1beta1` schema definition.
+2. The `useRegex` field will not be supported for insecure ingress/route when AKO is running in SNI mode or SNI Dedicated mode.
```

Comment:
Is saying `The `useRegex` field will not be supported for insecure Ingress/Route when AKO is running in SNI modes` not enough. ?

Reply:
Should be enough but better to specify the dedicated mode as well to avoid any kind of confusion.